app/javascript/app/pages/regions/regions-ghg-emissions/regions-ghg-emissions-styles.scss

```diff
@@ -5,15 +5,14 @@
   @include row();
 
   margin-top: $gutter-padding;
+  margin-bottom: $gutter-padding * 3;
 }
 
 .dropdowns {
   @include columns($columns: 12);
 
   @media #{$tablet-landscape} {
-    padding-left: $gutter-padding / 2;
-
-    @include columns($columns: (3.1,3.1,3.3,3));
+    @include columns($columns: (3,3,3.8,2.1));
```

Comment:
aand here as well, check the comments below 👇

Reply:
here it must stay in this way

Reply:
yeah, maybe in this case it's better this way 😁